Automattic/wp-calypso
# Reader profile - Add profile image

Related to https://github.com/Automattic/loop/issues/288

## Proposed Changes

* Add the user's avatar to the profile page.
* It does this by adding a fetch to the `/users/{user:id}/` endpoint and rendering the ReaderAvatar component.

## Why are these changes being made?


It's part of the profile page design.


## Testing Instructions



- Visit `/read/users/218803706` and ensure the page doesn't render.
- Visit `/read/users/218803706?flags=reader/user-profile` and make sure the profile page appears, along with the user avatar.
- Test other usernames with the following URL format: `/read/users/{userId}`

## Pre-merge Checklist
